src/python/CRABInterface/RESTBaseAPI.py

```diff
@@ -2,6 +2,8 @@
 import logging
 
 import cherrypy
+from cherrypy import request
```

Comment:
should be consistent in the file. Either import cherrypy and use cherrypy.request or import request from cherrypy and used is everywhere. I.e. either you stick with the style from the previous developer, or change it everywhere.